runtime/testy/Ansible.py

```diff
@@ -330,6 +329,12 @@ def unpackage(data):
         try:
             while True:
                 recv_data, addr = self.sock.recvfrom(2048)
+                if recv_data == b'':
```

Comment:
timeout should be included in the socket itself. When it returns an empty byte string, it indicates that the socket on the other end has closed so in this case, we would want to actually restart all ansible processes